pkgs/development/python-modules/getkey/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, flake8
+}:
+
+buildPythonPackage rec {
+  pname = "getkey";
+  version = "0.6.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0ng0ihfagh9g8hral0bq5nhjlp3csqghyv3z8b7ylkdkqc1cgiv8";
+  };
+
+  # disable coverage, because we don't care and python-coveralls is not in nixpkgs
+  postPatch = ''
+    sed -e '/python-coveralls/d' -e '/pytest-cov/d' -i setup.py
```

```suggestion
    substituteInPlace setup.py --replace python-coveralls "" --replace pytest-cov ""
```

The reason is that `substituteInPlace` will emit a warning if nothing was substituted, so we could some day drop this.

---

substituteInPlace does not have and equivalent for the d operator of sed. Anyway if nothing is substituted, it will not build, and anyway², getkey is not maintained so I doubt this will become problematic in the future.

---

Ah, yeah, you're right, I probably was awake for too long already X-D